homeassistant/components/overkiz/entity.py

```diff
@@ -93,14 +94,28 @@ class OverkizSensorDescription(SensorEntityDescription):
     ] | None = lambda val: val
 
 
+@dataclass
+class OverkizNumberDescriptionMixin:
+    """Define an entity description mixin for number entities."""
+
+    command: str
+
+
+@dataclass
+class OverkizNumberDescription(NumberEntityDescription, OverkizNumberDescriptionMixin):
+    """Class to describe an Overkiz number."""
+
+
 class OverkizDescriptiveEntity(OverkizEntity):
     """Representation of a Overkiz device entity based on a description."""
 
     def __init__(
         self,
         device_url: str,
         coordinator: OverkizDataUpdateCoordinator,
-        description: OverkizSensorDescription | ButtonEntityDescription,
+        description: OverkizSensorDescription
+        | ButtonEntityDescription
+        | OverkizNumberDescription,
```

Comment:
I think it would be better to use here the base class: `EntityDescription`
`from homeassistant.helpers.entity import EntityDescription`
```suggestion
        description: EntityDescription,
```

Reply:
Addressed in d8d7d3f